app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -20,6 +20,8 @@ def call
       private
 
       def access_token(input)
+        request_body = AuthnOidc::AuthenticateRequestBody.new(input.request)
```

Comment:
Do you think it should be a command class ? (because its only action is to parse the request )

Reply:
It's not really an action, it's more of an object representing the request body of the authn oidc request. You can look at `request_body.rb` (changed to `login_request_body.rb` in this PR) to see how we do it same for the login request.

In case the request will change and we will add more parameters to the input it will be easier if this will stay as a request object